tools/isobuild/package-source.js

```diff
@@ -422,18 +423,25 @@ _.extend(PackageSource.prototype, {
     utils.ensureOnlyExactVersions(options.cordovaDependencies);
     self.cordovaDependencies = options.cordovaDependencies;
 
-    var sources = _.map(options.sources, function (source) {
-      if (typeof source === "string")
-        return {relPath: source};
+    const sources = options.sources.map((source) => {
+      if (typeof source === "string") {
+        return {
+          relPath: source
+        };
+      }
+
       return source;
     });
 
-    var sourceArch = new SourceArch(self, {
+    const sourceArch = new SourceArch(self, {
       kind: options.kind,
       arch: "os",
       uses: _.map(options.use, splitConstraint),
-      getSourcesFunc: function () { return sources; },
-      nodeModulesPath: nodeModulesPath
```

Comment:
You're right! I thought it might have been a careless error.  Should have actually checked!  There's a var nodeModulesPath that's unused earlier in the function that you can kill as well.  Do that and LGTM.